streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -243,17 +243,22 @@ public void testGlobalThreadCloseWithoutConnectingToBroker() {
         props.setProperty(StreamsConfig.METRIC_REPORTER_CLASSES_CONFIG, MockMetricsReporter.class.getName());
         props.setProperty(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getPath());
         props.put(StreamsConfig.NUM_STREAM_THREADS_CONFIG, NUM_THREADS);
+        props.put("session.timeout.ms", 3001);
+        props.put(CommonClientConfigs.REQUEST_TIMEOUT_MS_CONFIG, 3002);
```

Comment:
why `3001` and `3002` ?

Reply:
There are dependencies among the timing configs, session timeout must be greater than some other timeouts. But this test doesn't need to take 3s.

I'm pushing a change to be clearer about what's going on here and also reduce the time to 200ms.